src/privatesend/privatesend-client.cpp

```diff
@@ -1561,8 +1561,8 @@ bool CPrivateSendClientSession::CreateDenominated(CAmount nBalanceToDenominate,
                 return fRegular || fFinal;
             };
 
-            // add each output up to 11 times or until it can't be added again or until we reach nPrivateSendDenoms
-            while (needMoreOutputs() && nOutputs <= 10 && currentDenomIt->second <= privateSendClient.nPrivateSendDenoms) {
+            // add each output up to 11 times or until it can't be added again or until we reach nPrivateSendDenomsBatched
```

Comment:
I think "batched" should probably be called goal or something, batched doesn't really make sense to me

Reply:
agree, a4f5b66